docs/index.md

```diff
@@ -100,7 +100,7 @@ Our API is setup at [//sandiegojs-vanilla-workshop.herokuapp.com][sdjs-app] and
 
 ## Getting started
 
-This project uses the build tool [Gulp][gulp] to automate a bunch of stuff for you. The gulp tasks are all defined in the `tasks` directory if you're the curious type. They are used to compile the sources and styles found within the `app` directory and put the final compiled source and styles into the `public` directory. The `public/index.html` will include the final styles and scripts inside of it.
+This project uses the build tool [Gulp][gulpjs.com] to automate a bunch of stuff for you. The gulp tasks are all defined in the `tasks` directory if you're the curious type. They are used to compile the sources and styles found within the `app` directory and put the final compiled source and styles into the `public` directory. The `public/index.html` will include the final styles and scripts inside of it.
```

Comment:
I think this should either be `[Gulp](http://gulpjs.com]` or remain as `[Gulp][gulp]` (`[Gulp][]` would work too) with a `[gulp]: http://gulpjs.com` added in this file.